source/includes/_people.md

```diff
@@ -1,6 +1,6 @@
-# People
+# PCO People
 
-Our People app helps you manage contact data, membership info, and everything you need to know about your people with confidence.
+PCO People helps you manage contact data, membership info, and everything you need to know about your people with confidence.
 
 ## Addresses
 
```

Comment:
This is auto-generated and will be overwritten on next build. To update this title and intro, update https://github.com/ministrycentered/people/blob/api/app/graphs/app_graph.rb.